src/translators/typescript.py

```diff
@@ -0,0 +1,748 @@
+from ast import literal_eval
+from cgitb import reset
+import enum
+from gc import is_finalized
+from json.encoder import py_encode_basestring
+from shutil import register_archive_format
+from src.ir import ast, typescript_types as tst, types as tp, type_utils as tu
+from src.transformations.base import change_namespace
+from src.ir.context import get_decl
+from src.translators.base import BaseTranslator
+
+def append_to(visit):
```

Comment:
This method is the same with that included in the kotlin.py file. Please add this method to a utility module and import from there.